README.rst

```diff
@@ -121,18 +121,6 @@ If you need support for other Google APIs, check out the
 
 .. _Google APIs Python Client library: https://github.com/google/google-api-python-client
 
-Quick Start
```

Comment:
Might make sense to replace this with instructions on installing specific service packages?

Reply:
I debated the same thing. In the moment I thought the lower list would be good enough but if you think there is value in adding a list of the GA `pip install _PACKAGE_` I think we should.

Reply:
True, let's leave it out for now.